pkg/api/rest/rest.go

```diff
@@ -125,6 +125,13 @@ func (w GracefulDeleteAdapter) Delete(ctx api.Context, name string, options *api
 	return w.Deleter.Delete(ctx, name)
 }
 
+// CollectionDeleter is an object that can delete a collection
+// of RESTful resources.
+type CollectionDeleter interface {
```

Comment:
Sorry - I don't understand this comment. What you would like to cleanup with it?